src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/PaymentMessageParser.java

```diff
@@ -14,6 +15,7 @@
 import java.util.List;
 
 @Service
+@Profile("!functional")
 public class PaymentMessageParser {
```

Comment:
please check the profiles mentioned in all places

Reply:
Apart from PayHubClient all other profiles look necessary.